src/taipy/gui_core/GuiCoreLib.py

```diff
@@ -278,9 +278,10 @@ class GuiCore(ElementLibrary):
                 "on_scenario_select": ElementProperty(PropertyType.function, f"{{{__CTX_VAR_NAME}.select_scenario}}"),
             },
         ),
-        "scenario_visualizer": Element(
+        "scenario": Element(
             "scenario",
             {
+                "id": ElementProperty(PropertyType.string),
```

Comment:
I'm not sure I see the point of this id (not dynamic, even).
Doesn't that reflect the scenario's id?

Reply:
I guess it can be used for styling (if assigned to the main element)

Reply:
Oh! The component id. That does make sense!
Thanks